packages/js/src/ui/components/primitives/Tooltip/TooltipRoot.tsx

```diff
@@ -28,7 +28,7 @@ export function TooltipRoot(props: TooltipRootProps) {
 
   const position = useFloating(reference, floating, {
     placement: props.placement || 'top',
-    strategy: 'absolute',
+    strategy: 'fixed',
```

Comment:
the `absolute` strategy doesn't work well for the tooltips, that's why changed to `fixed`